### DISCUSSION TITLE
What about Llama is?

### DISCUSSION THREAD

**Discussion by ivandzen** [alpaka-group/llama#190] [General]
Is this project related somehow to the article "Learning-based Memory Allocation for C++ Server Workloads"? In this article, author mentioned about memory allocator called Llama.

**Comment by bernhardmgruber** [comment]
Hi! This project is about the LLAMA as a "Low Level Abstraction of Memory Access". While my LLAMA can also integrate with allocators, such as the one you mentioned, it is not related to your article unfortunately. But you may still have a use for my LLAMA as well. You can find more in the documentation: https://llama-doc.rtfd.io/

**Reply by ax3l** [reply]
If you are looking for a memory allocator library, we also have one in our repertoire called mallocMC:
https://github.com/alpaka-group/mallocMC
